doc/Learn/Authentication/HowTo-MsalAuthentication.md

```diff
@@ -88,6 +88,35 @@ uid: Uno.Extensions.Authentication.HowToMsalAuthentication
     }
     ```
 
+    This configuration can also be done in the root App.cs file:
+    
+    ```csharp
+    private IHost Host { get; set; }
+
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var builder = this.CreateBuilder(args)
+            .Configure(host =>
+            {
+                host
+                .UseAuthentication(builder =>
+                {
+                    builder.AddMsal(msal =>
+                        msal
+                        .Builder(msalBuilder => 
+                            msalBuilder.WithClientId("161a9fb5-3b16-487a-81a2-ac45dcc0ad3b"))
+                        .Scopes(new[] { "Tasks.Read", "User.Read", "Tasks.ReadWrite" })
+                    );
+                });
+            });
+        ...
+    }
+    ```
+
+    > [!WARNING]
+    > A ClientId of GUID format is required for MSAL Authentication to work. You can specify it in the appsettings.json file, or in the code itself.
+    > If the ClientId cannot be found, the following warning will be logged: _Uno.Extensions.Authentication.MSAL.MsalAuthenticationProvider[0] A valid ClientId (GUID) is required for MSAL. Please provide the ClientId in the appsettings.json file._
```

Comment:
```suggestion
    > If the ClientId cannot be found, the following warning will be logged:
    > ```
    > _Uno.Extensions.Authentication.MSAL.MsalAuthenticationProvider[0] A valid ClientId (GUID) is required for MSAL. Please provide the ClientId in the appsettings.json file._
    > ```